### DISCUSSION TITLE
Why Neotree doesn't open pwd?

### DISCUSSION THREAD

**Discussion by saulchavezr** [LazyVim/LazyVim#1421] [Q&A] [answered]
![neotree_pwd_lazyvim](https://github.com/LazyVim/LazyVim/assets/14181836/e0f0cd33-7eaf-40f1-b2f8-ac1fb94fe2df)
The output of both `:pwd` and `:echo getcwd()` is `~/.config/nvim/lua/plugins`
However, both neotree and telescope act as if the current directory was `~/.config/nvim`, and it looks like it only happens with some directories. This is a fresh lazyVim install btw.
I also tried adding this option to neotree
```
  {
    "nvim-neo-tree/neo-tree.nvim",
    opts = {
      bind_to_cwd = true,
    },
  },
```
But it didn't work

**Comment by dpetka2001** [comment, marked as answer]
```lua
  {
    "nvim-neo-tree/neo-tree.nvim",
    opts = {
      filesystem = {      
		bind_to_cwd = true,
	  },
    },
  },
```
For the `cwd` this happens because Lazyvim uses the LSP to get the `root_dir` and uses that. If you would like to disable this behavior, then in your `~/.config/nvim/lua/config/lazy.lua` you should change to 
```lua
    {
      "LazyVim/LazyVim",
      init = function()
        require("lazyvim.util").get_root = function()
          return vim.loop.cwd()
        end
      end,
      import = "lazyvim.plugins",
    },
```

**Reply by saulchavezr** [reply]
Thanks, that worked perfectly
